PyPowerStore/provisioning.py

```diff
@@ -2109,6 +2109,47 @@ def delete_smb_share(self, share_id):
                                        self.server_ip, share_id))
     # SMB Share Methods End
 
+    # ACL Methods
+
+    def get_acl(self, share_id):
+        """
+        Retrieves the Access Control List (ACL) details for a given share ID.
+
+        :param share_id: The ID of the share for which to retrieve the ACL details.
+        :type share_id: str
+        :return: The response from the client's request to retrieve the ACL details.
+        :rtype: dict
+        """
+        LOG.info("Getting ACL details: '%s'" % share_id)

```

Comment:
can we use f-string format here for the logging?

Reply:
Yes, updated